### PR TITLE
feat(controller): expand random namespace for apps

### PR DESCRIPTION
**Pull request by mboersma**
More 8-letter nouns and 6-letter adjective-ish words! :books:

### CODE REVIEW

File: controller/api/utils.py [hunk 1]
```diff
@@ -9,31 +9,31 @@
...
+        'gaslit', 'gentle', 'gifted', 'ginger', 'global', 'golden', 'grassy', 'guided',
+        'haptic', 'hearty', 'hidden', 'hipper', 'honest', 'humble', 'hungry', 'hushed',
+        'iambic', 'iciest', 'iconic', 'indoor', 'inward', 'ironic', 'island', 'italic',
+        'jagged', 'jangly', 'jaunty', 'jicama', 'jiggly', 'jovial', 'joyful', 'junior',
+        'kabuki', 'karmic', 'keener', 'kiddie', 'kindly', 'kingly', 'klutzy', 'knotty',
+        'lambda', 'latest', 'leader', 'linear', 'lively', 'lonely', 'loving', 'luxury',
+        'madcap', 'madras', 'marble', 'mellow', 'metric', 'modest', 'molten', 'mystic',
+        'native', 'nearby', 'nested', 'newish', 'nickel', 'nimbus', 'nonfat', 'normal',
+        'oblong', 'oddest', 'offset', 'oldest', 'onside', 'orange', 'outlaw', 'owlish',
+        'padded', 'pastry', 'peachy', 'pepper', 'player', 'preset', 'proper', 'pulsar',
```
Comment (carmstrong) [inline controller/api/utils.py]: Isn't pastry a noun? ;)

**Comment by carmstrong** [comment]
Code LGTM.

File: controller/api/utils.py (hunk 1 above)
Comment (bacongobbler) [inline controller/api/utils.py]: same for kiddie and vacuum. walkup looks like a funny one. Where are you getting these words from, @mboersma? :)

**Comment by bacongobbler** [comment]
Asides for my comment, code LGTM! Can't wait for `scenic-landmass` to come around. :D

File: controller/api/utils.py (hunk 1 above)
Comment (mboersma) [inline controller/api/utils.py]: I just google for "8-letter words starting with a," for example. I ignore anything negative or edgy. If those ones are too clunky as adjective-type words I can look for others.

**Pull request merged**